server/grpc_framework_server.go

```diff
@@ -73,8 +73,13 @@ func NewGrpcFrameworkServer(config *ServerConfig) (*GrpcFrameworkServer, error)
 	for issuer := range config.Security.Authenticators {
 		log.Infof("Authentication enabled for issuer: %s", issuer)
 
-		// Check the necessary security config options are set
-		if config.Security.Role == nil && (config.AuthZUnaryInterceptor == nil || config.AuthZStreamInterceptor == nil) {
+		// Check the necessary security config options are set. Authentication is enabled. Therefore,
+		// either RoleManager must be provided (this implies use of the default authZ)
+		// or explicit authZ interceptors must be provided
+		// or external authZ checker must be proviced (this implies use of external_authorizer.go)
+		if config.Security.Role == nil &&
+			(config.AuthZUnaryInterceptor == nil || config.AuthZStreamInterceptor == nil) &&
+			config.ExternalAuthZChecker == nil {
 			return nil, fmt.Errorf("must supply role manager when authentication is enabled and default authZ is used")
```

Comment:
change the error message?

Reply:
@adityadani : Seems like the error message is still correct. What would you like to be changed?